.github/workflows/build_and_test_rpms.yml

```
@@ -0,0 +1,25 @@
+name: Build and test RPMs
+on: [pull_request]
+
+jobs:
+  build-rpms-and-test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Prepare Docker
+        run: |
+          echo 'DOCKER_OPTS="-H tcp://127.0.0.1:2375 -H unix:///var/run/docker.sock -s devicemapper"' | sudo tee /etc/default/docker > /dev/null &&
+            sudo service docker restart
+
+      - name: Build RPMs
+        run: |
+          docker run -v $(pwd):/${GITHUB_REPOSITORY##*/} \
+                     -v $(pwd)/_build_dir:/u/_build_diir \
```

```suggestion
          docker run -v "$PWD:/${GITHUB_REPOSITORY##*/}" \
                     -v "$PWD/_build_dir:/u/_build_dir" \
```

- minor: is there a reason for `$(pwd)` (invoking the `pwd` command) vs just using `$PWD` ?
- quotes: probably fine without these assuming the workflow environment plops you down into a safe-looking path, but otherwise generally it's a good idea to quote working dir when you don't set it to a known value yourself
- `_build_diir`: did i catch a Niner in there?